User signup input

brandonhumphries opened the pull request:
new user can now signup with an email and password, and the user email and password are captured and just logged to console at this point. Will eventually be posting to our server. @hglasser @TRACYMUSIKER Can you please take a look when you get a chance? Thank you!

TRACYMUSIKER commented on index.html:
```diff
@@ -26,6 +29,22 @@
+            <div class="signup-input-container">
+                <form class="signup-form">
+                    <h3 class="signup-header">
+                        Are you new to In-Your-Fridge? Sign up and create an account!
```
and maybe lowercase, but looks great!

brandonhumphries commented:
updated header to reflect app name. also, updated the class names for login email and password so that we can have one js function that can be called in either login or signup submission. @hglasser @TRACYMUSIKER can you please review? Thank you!
